### PR TITLE
fix(README): remove redundant examples and cleanup

### LINKED ISSUE #80: Refactor README
Hey guys,
there are a lot of extra examples in  "README". in many validating functions, 1 or 2 examples is enough. I think we should remove extra examples to make "README" more clear.

### PR DESCRIPTION
**Pull request by masoudDaliriyan**

### CODE REVIEW

File: README.md
Comment (mediv0) [inline README.md]: this misspell is intentional to show how the fuzzy option works
you need to revert this

**Review by mediv0 — CHANGES_REQUESTED** [review]

File: README.md
Comment (masoudDaliriyan) [reply README.md]: Sorry, My bad.
I'll fix it today, but there were test errors in the project.
Why the master branch must have the test errors? 
@ali-master  @mediv0

Comment (masoudDaliriyan) [reply README.md]: @mediv0 Do you think one mistake is enough to show how fuzzy options works? Or I Must revert it?

Comment (mediv0) [reply README.md]: > @mediv0 Do you think one mistake is enough to show how fuzzy options works? Or I Must revert it?

maybe one is enough

Comment (mediv0) [reply README.md]: > Sorry, My bad.
> I'll fix it today, but there were test errors in the project.
> Why the master branch must have the test errors?
> @ali-master @mediv0

tests are failing because of previous commits, not yours
we can merge this after @ali-master approves your changes

Comment (ali-master) [reply README.md]: Please revert this line's changes.

**Comment by ali-master** [comment]
Please revert that line which I and @mediv0 said, and then it will be ready to merge.

**Comment by masoudDaliriyan** [comment]
@ali-master This pull Request is editable by maintainers. I'll do that but maybe this feature is useful.
Thank you for your response.

**Comment by mediv0** [comment]
@ali-master I fixed fuzzy typo
rdy to merge

**Pull request merged**
